.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a misbehaviour in the system
+
+---
```

Comment:
What about meta info like environemnt (OS, software version/git commit, local weather)?

Reply:
You forget the local goblin in your laptop that makes the issue non-deterministic! :trollface:

Reply:
@adinapoli-mndc It might be a goblin, but it wants to be called by its name: **works-for-me**.